### ISSUE TITLE
Suggestion to add provider support for easyDNS

### ISSUE THREAD

**Issue by txr13** [AnalogJ/lexicon#20]
I would like to suggest adding support for easyDNS as a provider to lexicon. Documentation is available at http://docs.sandbox.rest.easydns.net/, and the signup for an API key is located at http://docs.sandbox.rest.easydns.net/beta_signup.php

I am not fluent in Python (hence the lack of a pull request), but I am very willing to help test code using my own easyDNS account / domains. I'm particularly interested in this for use with Let's Encrypt's DNS challenges.

Thanks for considering this!

**Comment by AnalogJ** [comment]
Hey @txr13 I'm more than happy to add support for easyDNS, I'll take a look at their api and get back to you.

**Comment by AnalogJ** [comment]
Hey @txr13 can you try v1.0.22 of the lexicon library? I've gotten a easydns api key, and I was able to test against the sandbox environment.

**Comment by txr13** [comment]
Testing with lexicon 1.0.23:

First I ran into a big, fat 403 Forbidden. After basic testing with curl to make sure I didn't mistype the API token/key (user/pass substitute), I realized that lexicon has the live API endpoint, not the sandbox. Edited easydns.py to fix that for testing.

List of all TXT records succeeded.
Create TXT records succeeded.
Delete TXT records by name/content succeeded.
Delete TXT records by identifier succeeded.

Looks good so far to me! I'll see about moving off the sandbox and into the live environment, then trying it out with letsencrypt.sh. Will report back after.
